data/core/macros/special-notes.cfg

```diff
@@ -113,6 +113,12 @@ _"This unit has a defense cap on certain terrain types — it cannot achieve a h
     [/special_note]
 #enddef
 
+#define NOTE_ABILITY_REMOVE ABILITY
+    [+{ABILITY}]
+        special_note=""
```

Comment:
On the White Mage's and Elvish Druid's help pages, there's a blank bullet point where the second `[heals]` ability's special note should go.

Note: the `special_note=""` of the NOTE_ABILITY_REMOVE macro is also cut&pasted without using the macro into somewhere in abilities.cfg.

Reply:
This was fixed in the next push.